pyleecan/Methods/Mesh/MeshMat/save_vtk.py

```diff
@@ -0,0 +1,42 @@
+import os
+from ....Functions.GUI.log_error import log_error
+
+
+def save_vtk(self, save_path):
+    """Save mesh to a .vtk file
+
+    Parameters
+    ----------
+    self : Mesh
```

Comment:
We already know how to save vtk in Manatee => Ask Hélène how we save the mesh in Manatee
add save_vtk method to MeshVTK (if needed)